src/component/TitleBar/TitleBar.tsx

```diff
@@ -1,11 +1,13 @@
 import * as React from 'react';
-
+import * as classNames from 'classnames/bind';
+import { Node } from '../../flowTypes';
 import * as styles from './TitleBar.scss';
 
 interface TitleBarProps {
     title: string;
+    node: Node;
     onRemoval(event: React.MouseEvent<HTMLDivElement>): any;
-    className?: string;
+    __className?: string;
```

Comment:
why move to dunder? what's the convention?

Reply:
we've sort of already set that convention elsewhere, the idea being that it could be confused w/ React's `className` property. borrowed it from EventBrite's style guide: https://github.com/eventbrite/javascript/tree/master/react#classname-prop

Reply:
I'm fine with it, just want to understand so we can aim for consistency. Maybe we should draw up our own conventions.

Reply:
ya that would be awesome, if we need inspiration the airbnb style guide is pretty popular.

Reply:
will take a look and see if there's anything else we're already doing so we can  discuss 👍

Reply:
I'm fine adopting an existing one and making any amendments from there if we feel strongly enough to deviate. No reason to waste a ton of time on it, just having an understanding is the useful bit.

Reply:
ok sounds good, if like airbnb's style i can add a config to tslint. also, in case you haven't already installed it, the tslint vscode plugin will show you linting errors so that might be helpful

Reply:
Yup, in a separate PR.